htdocs/user/class/user.class.php

```diff
@@ -3298,14 +3298,21 @@ public static function replaceThirdparty(DoliDB $db, $origin_id, $dest_id)
 	public function load_state_board()
 	{
 		// phpcs:enable
+	    global $conf;
 
 		$this->nb = array();
 
-		$sql = "SELECT count(u.rowid) as nb";
+		$sql = "SELECT DISTINCT count(u.rowid) as nb";
```

Comment:
What means a DISTINCT COUNT() ?
You means   COUNT(DISTINCT  ) ?

Reply:
thank you